esp/src/src/nls/es/hpcc.ts

```diff
@@ -350,6 +357,7 @@ export = {
     FileUploadStillInProgress: "Carga de archivos todavia esta progresando",
     Filter: "Filtro",
     FilterDetails: "Detalles del filtro",
+    FilterMetricsTooltip: "Esto filtrará según la columna 'Ámbito' de forma predeterminada. Para filtrar en cualquier otra columna, use el nombre de la columna como prefijo en su búsqueda, por ejemplo: 'NombreArchivo:spill'",
```

Comment:
I would inject "basado" between "filtrar" and "en cualquier"

"Esto filtrará según la columna 'Ámbito' de forma predeterminada. Para filtrar basado en cualquier otra columna, use el nombre de la columna como prefijo en su búsqueda, por ejemplo: 'NombreArchivo:spill'",